src/network.rs

```diff
@@ -85,14 +85,23 @@ impl Network {
         )
     }
 
-    pub fn section_size_dist(&self) -> Distribution {
-        Distribution::new(self.sections.values().map(
+    pub fn age_aggregator(&self) -> Aggregator {
+        Aggregator::new(
+            self.sections
+                .values()
+                .flat_map(|section| section.nodes().values())
+                .map(|node| u64::from(node.age())),
+        )
+    }
+
+    pub fn section_size_aggregator(&self) -> Aggregator {
+        Aggregator::new(self.sections.values().map(
             |section| section.nodes().len() as u64,
```

Comment:
does this need to change to `u64::from` as well?

Reply:
I didn't get any warning here, so not sure. Maybe it's from different clippy version?

Reply:
I just tried it. There is no `impl From<usize> for u64`.